frappe/templates/emails/newsletter.html

```diff
@@ -0,0 +1,10 @@
+<div style="text-align: justify; padding-left: 150px;">
+    <div style="width: 600px;">
+        {{message}}
+    </div>
+</div>
```

Comment:
```suggestion
<div style="text-align: justify;">
    <div style="width: 600px; margin: 10px auto;">
        {{ message }}
    </div>
</div>
```
you can use `margin: auto` to center the content, instead of hardcoding padding value.